radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/substatehash/BerkeleySubstateAccumulatorHashStore.java

```diff
@@ -510,6 +515,10 @@ private Writer openEpochsHashFileAsAppend() {
     }
   }
 
+  private String getEpochHashFilenameForCurrentNetwork() {
+    return EPOCH_HASH_FILENAME + "." + this.currentNetwork.name().toLowerCase();
```

Comment:
I'd be tempted to end this with `.txt` to save confusing programs which use the file ending to detect file types :)

Reply:
so epoch-hash.mainnet.txt for exemple?

Reply:
Yup

Reply:
done